Vendor/macOS/MSAL/MSAL.framework/Versions/A/Headers/MSALPublicClientApplication.h

```diff
@@ -225,7 +225,8 @@
  */
 - (void)allAccountsFilteredByAuthority:(nonnull MSALAccountsCompletionBlock)completionBlock;
 
-#pragma SafariViewController Support
+#pragma mark -
```

Comment:
nit: Is `pragma mark -` needed?

Reply:
That's the fix for the issue that we have. 😄